.github/workflows/insider-linux.yml

```diff
@@ -279,6 +279,8 @@ jobs:
         - slug: LOONG64
           vscode_arch: loong64
           npm_arch: loong64
```

Comment:
I missed the introduction of the `slug` field when rebasing my changes to master earlier today. I will add a slug with the next update of the PR.